Orange/widgets/classify/owclassificationtree.py

```diff
@@ -65,6 +65,26 @@ def get_learner_parameters(self):
             ("maximum depth {}".format(self.max_depth), self.limit_depth)) if c) or "None"
         return items
 
+    def init_code_gen(self):
+        def run():
+            ow.set_data(input_data)
```

Comment:
Classification Tree should converted to code in the same way as File (no widget, just library code). What the widget does is just calls ClassificationTree class with selected parameters to construct a learner and then calls the learner with input data to get a classifier.

If learners implemented __repr__ in a way that the returned string is a valid python code that constructs a learner, this could be simplified further.